source/docs/http3_upstream.md

```diff
@@ -1,7 +1,17 @@
 ### Overview
 
 Support for Upstream HTTP/3 connections is slightly more complex than for HTTP/1 or HTTP/2
-and this document attempts to describe it.
+and requires specific configurations to enable it. HTTP/3 is only attempted to servers which
```

Comment:
If HTTP/3 is configured in auto_config, HTTP/3 connections will only be attempted

Since it is force-attempted for explicit_http_config and "do what downstream did"

Reply:
Good point. Done.